zips/zip-0325.md

```diff
@@ -171,16 +171,21 @@ valid hardened child index) within its key tree for this purpose.
   most 252 bytes that identifies the desired private-use context.
 - Return $\mathsf{CKDreg}(K, \mathtt{0x7FFFFFFF}, \mathsf{PrivateUseSubject})$
 
-:::warning
+<div class=warning></div>
```

Comment:
This affects the following paragraph. At some point I'll fix it to support multi-paragraph notes/warnings, but for now this was the only way I could get the CSS to work.